themes/default/content/docs/pulumi-cloud/oidc/_index.md

```diff
@@ -51,26 +66,37 @@ Valid custom claims for this service are listed in the table below:
 
 ### Pulumi ESC
 
-The format of the subject claim for this service is:
+The default format of the subject claim for this service is:
 
 `pulumi:environments:org:<organization name>:env:<environment name>`
 
+However, we recommend using `subjectAttributes` property to customize the OIDC subject claims if you plan to use subject claims in your cloud provider trust policy. The default prefix when using `subjectAttributes` will be
```

Comment:
While this is a powerful feature, a lot of people are able to work fine with the existing format. I would not use the word "recommend"